go/vt/wrangler/schema.go

```diff
@@ -98,7 +98,12 @@ func (wr *Wrangler) ReloadSchemaShard(ctx context.Context, keyspace, shard, repl
 			defer wg.Done()
 			concurrency.Acquire()
 			defer concurrency.Release()
-			if err := wr.tmc.ReloadSchema(ctx, tablet, replicationPos); err != nil {
+			pos := replicationPos
+			// Master is always up-to-date. So, don't wait for position.
+			if tablet.Type == topodatapb.TabletType_MASTER {
+				pos = ""
+			}
+			if err := wr.tmc.ReloadSchema(ctx, tablet, pos); err != nil {
```

Comment:
This seems like an unrelated change to the PR description -- I _assume_ this is a separate bug which triggered the whole sweeping error reporting change, but if so it should either be called out explicitly in the PR or (better IMO) pulled out into a separate bug fix change and tested.

Reply:
The only change in behavior is that the annoying warnings don't get printed, which I tested manually. As for the PR, the description calls out this change, and it's a separate commit. But I can pull this out in a separate PR if you insist.

I'll address the other comments momentarily.